frontend/src/App.tsx

```diff
@@ -1,13 +1,13 @@
 import '@bcgov/design-tokens/css/variables.css';
 import '@bcgov/bc-sans/css/BC_Sans.css';
+import LandingPage from './Views/LandingPage';
 import Header from './Commons/Header';
 import Footer from './Commons/Footer';
-import Button from './Commons/Button';
 
 const App = () => (
   <>
     <Header />
-    <Button text="New Calculation" />
+    <LandingPage />
     <Footer />
   </>
 );
```

Comment:
Nice! I might need to figure out how to work this for my current PR. I added a new page for the main calculation form. Not sure if that's correct. We'll see soon enough.

Reply:
Yeah I'll check it out once you fired your pr